src/content/docs/en/install-and-setup.mdx

```diff
@@ -398,12 +398,6 @@ If you prefer not to use our automatic `create astro` CLI tool, you can set up y
     }
     ```
 
-    Finally, create `src/env.d.ts` to let TypeScript know about ambient types available in an Astro project:
```

Comment:
Why do we suggest no longer creating this file vs jsut updating the content, if we still show it in places like above and it still ships in our basic examples?

Reply:
Because we generate it for the user so I don't think they need to do it manually? Also this file will likely become optional in 5.0